hivemq-edge/src/frontend/src/api/hooks/useGateway/useGetListeners.tsx

```diff
@@ -6,8 +6,11 @@ import { ApiError, ListenerList } from '@/api/__generated__'
 export const useGetListeners = () => {
   const appClient = useHttpClient()
 
-  return useQuery<ListenerList, ApiError>([QUERY_KEYS.LISTENERS], async () => {
-    const item = await appClient.gatewayEndpoint.getListeners()
-    return item
+  return useQuery<ListenerList, ApiError>({
+    queryKey: [QUERY_KEYS.LISTENERS],
+    queryFn: async () => {
+      const item = await appClient.gatewayEndpoint.getListeners()
+      return item
```

Comment:
suggestion(non-blocking): can be returned directly and aligns with other implementation within the PR

```suggestion
      return await appClient.gatewayEndpoint.getListeners()
```